Failed to renovate lerna monorepo in  Gitlab Ci

**What Renovate type are you using?**
Renovate CLI

**Describe the bug**
Running renovate cli in Gitlab Ci to renovate a lerna monorepo, result in `Artifact update problem`

```
### :warning: Artifact update problem

Renovate failed to update an artifact related to this branch. You probably do not want to merge this MR as-is.

:recycle: Renovate will retry this branch, including artifacts, only when one of the following happens:

 - any of the package files in this branch needs updating, or 
 - the branch becomes conflicted, or
 - you check the rebase/retry checkbox if found above, or
 - you rename this MR's title to start with "rebase!" to trigger it manually

The artifact failure details are included below:

##### File name: durian/package-lock.json

```
lerna notice cli v3.13.3
lerna info ci enabled
lerna info Bootstrapping 5 packages
lerna info Installing external dependencies
lerna ERR! npm ci --package-lock-only --no-audit exited 1 in 'durian'
lerna ERR! npm ci --package-lock-only --no-audit stderr:
npm ERR! cipm can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
npm ERR! 
npm ERR! 
npm ERR! Invalid: lock file's @types/express@4.16.0 does not satisfy @types/express@4.16.1
npm ERR! 

npm ERR! A complete log of this run can be found in:
npm ERR!     /tmp/renovate_cache/others/npm/_logs/2019-04-19T10_54_34_343Z-debug.log

lerna ERR! npm ci --package-lock-only --no-audit exited 1 in 'durian'
lerna WARN complete Waiting for 3 child processes to exit. CTRL-C to exit immediately.

```


```


**To Reproduce**
Steps to reproduce the behavior:

1.  Create a lerna monorepo 
2.  Run renovate cli in gitlab ci to renovate it

**Additional context**

Adding `--no-ci` when running lerna should solve this ( https://github.com/lerna/lerna/tree/master/commands/bootstrap#--no-ci ).

--- a/lib/manager/npm/post-update/lerna.js
+++ b/lib/manager/npm/post-update/lerna.js
@@ -43,9 +43,9 @@ async function generateLockFiles(
       params =
         '--ignore-scripts --ignore-engines --ignore-platform --mutex network:31879';
     }
-    cmd = `npm i -g -C ~/.npm/lerna@${lernaVersion} lerna@${lernaVersion} && ${lernaClient} install ${params} && ~/.npm/lerna@${lernaVersion}/bin/lerna bootstrap -- ${params}`;
+    cmd = `npm i -g -C ~/.npm/lerna@${lernaVersion} lerna@${lernaVersion} && ${lernaClient} install ${params} && ~/.npm/lerna@${lernaVersion}/bin/lerna bootstrap --no-ci -- ${params}`;
     if (binarySource === 'global') {
-      cmd = `${lernaClient} install ${params} && lerna bootstrap -- ${params}`;
+      cmd = `${lernaClient} install ${params} && lerna bootstrap --no-ci -- ${params}`;
     }
     logger.debug({ cmd });
     // TODO: Switch to native util.promisify once using only node 8

